src/js/providers/video-listener-mixin.js

```diff
@@ -37,6 +37,9 @@ const VideoListenerMixin = {
 
     timeupdate() {
         this.stopStallCheck();
+        if (this.seeking) {
+            return;
+        }
```

Comment:
Let's move this to after `this._helperLastVideoHeight = height` so we still get adaption (visual quality) events.